content/en/docs/contribute/style/style-guide.md

```diff
@@ -46,20 +46,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
 
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. 
```

Comment:
Hi @geoffcline .
After reading through the changes, I think it would be worthwhile to focus on simplicity
and try to make small changes to move this forward.
Any guidance that is not explicit requires more thought to put into action.
I think the main point of this change is:
```You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence.```
I think you could remove the other sentences about capitalization and change this sentence as you see fit.
I suggest to remove the example sentences and instead push these examples into the tables (without
the dialog about what is right or wrong). Show by example.

Reply:
I sincerely appreciate your guidance on making small changes to move this forward :)